contacts.html

```diff
@@ -90,7 +90,7 @@ <h3 class="header center green-text">Contact us</h3>
                         <div class="response" id="mce-success-response" style="display:none"></div>
                     </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
                     <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_83a95d4fc93cca6797b6b6832_208855fbfd" tabindex="-1" value=""></div>
-                    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button"></div>
+                    <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe" class="button" style="margin:0.5em 1em"></div>
```

Comment:
would be better if you do it in `style.css` file, so we do not touch 3d party code.